src/structures/APIMessage.js

```diff
@@ -202,8 +198,7 @@ class APIMessage {
       content,
       tts,
       nonce,
-      embed: !isWebhookLike ? (this.options.embed === null ? null : embeds[0]) : undefined,
-      embeds: isWebhookLike ? embeds : undefined,
+      embeds,
```

Comment:
```suggestion
      embeds: embeds.length ? embeds : undefined
```
This prevents stripping embeds when not provided

Reply:
Isn't that how it worked before this change? It could only be removed by passing `null`.

Reply:
And this ensures it will continue to work that way.

Reply:
Made me realize that sometimes we want to send an empty array though...fixed the comment to address that

Reply:
Should it be `typeof this.options.embeds === 'undefined'`? This will treat explicitly passing `null` as falsey

Reply:
`null` isn't actually a valid type to pass currently, if that wants to be supported then yes it should be typeof